README.md

```diff
@@ -21,3 +21,18 @@ The library is distributed using a modified BSD-style
 
 The official source code repository is located at
 https://github.com/avrdudes/avr-libc/
+
+### Building AVR-LibC from the Source Code
+
+AVR-LibC depends on [GNU Binutils](https://sourceware.org/binutils/) (>= 2.13)
+and [GCC](https://gcc.gnu.org/) (>= 3.3) that should be built for the AVR
+target (configure --target=avr). Detailed instructions how to build these from
```

Comment:
This gives the false impression that something like `./configure --target=avr` should be used or would work. In fact, according to GCC install instructions, GCC does not support configure anywhere in the source tree

Reply:
I removed the direct reference to `./configure --target=avr`. It is a technical detail that should be left to a specialized documentation page.